query-engine/prisma-models/tests/datamodel_converter_tests.rs

```diff
@@ -353,17 +284,6 @@ fn explicit_relation_fields() {
     let post = datamodel.assert_model("Post");
     let relation = datamodel.assert_relation((blog.id, post.id), relation_name);
 
-    blog.assert_relation_field("posts")
-        .assert_list()
-        .assert_optional()
-        .assert_relation_name(relation_name)
-        .assert_side(RelationSide::A);
-
-    post.assert_relation_field("blog")
-        .assert_optional()
-        .assert_relation_name(relation_name)
-        .assert_side(RelationSide::B);
```

Comment:
All of this is not handled by prisma-models anymore, so there's little point in testing it here.